app/config/services.yml

```diff
@@ -14,12 +14,22 @@ services:
         # if you need to do this, you can override this setting on individual services
         public: false
 
-    # loads services from whatever directories you want (you can add directories!)
+    # makes all classes in src/AppBundle available as services
     # this creates a service per class whose id is the fully-qualified class name
     AppBundle\:
-        resource: '../../src/AppBundle/{Command,Form,EventSubscriber,Twig,Security}'
+        resource: '../../src/AppBundle/*'
```

Comment:
It definitely does - the `FileLoader` always passes a `$recursive` flag set to `true` to the glob functionality. But, I totally understand what you're referring to with the normal `**` syntax :)